packages/xo-web/src/xo-app/settings/logs/index.js

```diff
@@ -99,7 +99,7 @@ const ACTIONS = [
 
 const INDIVIDUAL_ACTIONS = [
   {
-    handler: log => alert(_('logError'), <Copiable tagName='pre'>{formatLog(log)}</Copiable>),
+    handler: log => alert(_('logError'), <Copiable tagName='pre'>{uuidToLink(formatLog(log))}</Copiable>),
```

Comment:
Don't you need to pass `data` explicitly as a prop to `Copiable` since its `children` is not longer a simple string?